doc/rest-api.md

```diff
@@ -2285,19 +2285,33 @@ Input:
         "type": "custom"
     }
 
+Input:
```

Comment:
Should be:

    Input (copying a volume):

Reply:
added